suites/quincy/rbd/tier-2_rbd_mirror_regression.yaml

```diff
@@ -143,93 +143,94 @@ tests:
       module: exec.py
       name: configure mon_allow_pool_delete to True
 
-  - test:
-      name: test-image-delete-from-primary-site
-      module: test-image-delete-primary-site.py
-      clusters:
-        ceph-rbd1:
-          config:
-            imagesize: 2G
-            io-total: 200M
-      polarion-id: CEPH-9501
-      desc: Verify that image deleted at primary site updated at secondary
+  # - test:
```

Comment:
why all these tests got commented out?

Reply:
sorry i was testing one change to only this particular test case ....un-commented all .